### PR TITLE
feat(NN-1): update get list

### PR DESCRIPTION
**Pull request by road2global**

### CODE REVIEW

File: server/pkg/models/menu/menu.py [hunk 1]
```diff
@@ -6,8 +6,11 @@ def __init__(self, name, params):
         self.name = name
         self.params = params
 
-    def get():
-        pass
+    def get(self, name):
```
Comment (din-mukhammed) [inline server/pkg/models/menu/menu.py:9]: name is already declared in self
```
def get(self):
```

Comment (road2global) [reply server/pkg/models/menu/menu.py:9]: This is drugoi name

File: server/pkg/models/menu/menu.py [hunk 2]
```diff
@@ -6,8 +6,11 @@ def __init__(self, name, params):
...
+        doc = db.collection(u'menu').document(u'{name}').format(name=name)
+        if doc.exists:    
+            return (f'{name} ==> {doc.get().to_dict()}'), 200
+        return "Data not found", 404
```
Comment (din-mukhammed) [inline server/pkg/models/menu/menu.py:13]: we don't need to return http status codes. We should fill Menu `self.params` here
```
if doc.exists:
      self.params = doc.get().to_dict()
   
```

Comment (din-mukhammed) [reply server/pkg/models/menu/menu.py:13]: Or self.data = doc.get().to_dict()

File: server/web/v1/menu/list.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 from . import menu_handler
-
+from pkg.models.menu.menu import Menu
 
 @menu_handler.route("/list")
 def list():
-    return "list"
+    return Menu("id", "1").get("id")
```
Comment (din-mukhammed) [inline server/web/v1/menu/list.py:6]:
```
Menu("id", "1").get()
return Menu.data
```

File: server/pkg/models/menu/menu.py (hunk 1 above)
Comment (din-mukhammed) [reply server/pkg/models/menu/menu.py:9]: or may be better to name it: `load(self)`

**Pull request merged**